bin/cml-send-comment.js

```diff
@@ -11,7 +11,7 @@ const run = async (opts) => {
   const path = opts._[0];
   const report = await fs.readFile(path, 'utf-8');
   const cml = new CML(opts);
-  await cml.commentCreate({ ...opts, report });
+  console.log(await cml.commentCreate({ ...opts, report }));
```

Comment:
should not have a print statement? Why are we printing in stderror?

Reply:
Nice catch!